Gemfile.lock

```diff
@@ -426,6 +476,7 @@ DEPENDENCIES
   tzinfo-data
   uglifier (>= 1.3.0)
   web-console (>= 3.3.0)
+  with_advisory_lock
```

Comment:
do not mind the position of this comment. What is the purpose of this gemfile.lock file?